.github/workflows/ci.yml

```diff
@@ -40,4 +40,4 @@ jobs:
               run: TESTSUITE=integration MINCOVERAGE=85 ci/test.sh
 
             - name: Execute Cache tests
-              run: /vendor/bin/phpunit --testsuite=cache --stop-on-fail
+              run: ./vendor/bin/phpunit --testsuite=cache --stop-on-fail
```

Comment:
I was fiddling around with this since `cache` testsuite wasn't being recognized, and it was passing, please double check that before merging.

Reply:
Ah OK. So following your comment, if I look at https://github.com/supportpal/api-client-php/pull/52/commits/5775a06c95f92754567907a5d441416ede87e6fb you can see it's not a problem with calling phpunit, so the removal of `.` won't help. Looks like an issue with `--testsuite=` or `phpunit.xml`

```
Run ./vendor/bin/phpunit --testsuite=cache --stop-on-fail

PHPUnit 8.5.8 by Sebastian Bergmann and contributors.

No tests executed!
```

Reply:
Yes, not sure where the issue is. I checked it very briefly,  `./vendor/bin/phpunit --testsuite=cache` works on my local just fine

Reply:
See https://github.com/sebastianbergmann/phpunit/issues/4493